src/main/java/com/bootcamp/demo/model/Promotion.java

```diff
@@ -0,0 +1,37 @@
+package com.bootcamp.demo.model;
+
+public class Promotion {
+    public String getCode() {
+        return code;
+    }
+
+    private String code;
+    private int amount;
+    private boolean isFlatAmount;
```

Comment:
what's the purpose of this field?

Reply:
so that the dicount could be either flat or a percentage

Reply:
I understand, but probably we should have an enum here also rather than a boolean